src/Features/Core/Portable/SemanticSearch/Tools/ReferencingSyntaxFinder.cs

```diff
@@ -0,0 +1,124 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if NET6_0_OR_GREATER
```

Comment:
curious why NET6, not just NET.

Reply:
Copied from another file. Can clean up later.